pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1407,7 +1407,7 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claim *v1.Persis
 		strerr := fmt.Sprintf("Mount options are not supported by the provisioner but StorageClass %q has mount options %v", storageClass.Name, options.MountOptions)
 		glog.V(2).Infof("Mount options are not supported by the provisioner but claim %q's StorageClass %q has mount options %v", claimToClaimKey(claim), storageClass.Name, options.MountOptions)
 		ctrl.eventRecorder.Event(claim, v1.EventTypeWarning, events.ProvisioningFailed, strerr)
-		return
+		return fmt.Errorf("provisioner %q doesn't support mount options", plugin.GetPluginName())
```

Comment:
@jsafrane, could you verify that this is correct?

Reply:
Yes, it's correct